README.md

```diff
@@ -1,116 +1,168 @@
-This is a readme for the RETNET team
+# Are You Using Retentive Networks? (RetNets) 📝
```

Comment:
Do we want the header to be the name of our paper? When I search up the paper name, this pops up in search results. I am not sure if this is anonymous enough.

Reply:
I searched the whole title and a few variants in Google, but didn't get anything back on the 1st page of results in either Incognito nor a normal browsing session. I wonder if this is a cookies thing specific to those using the repo frequently by searching that name? Try looking it up in an Incognito browser.

Reply:
I get responses on DuckDuckGo and Bing, but not Google.

Reply:
Interesting. I still get nothing on Edge and Firefox. I guess we can change the title name for now. The paper title is more likely than not to undergo changes as well, given we're doing something of a survey now, but I'll put in a place holder.